flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfigurationParserFactory.java

```diff
@@ -105,13 +104,13 @@ private int getRestPort(CommandLine commandLine) throws FlinkParseException {
 		}
 	}
 
-	private static JobID getJobId(CommandLine commandLine) throws FlinkParseException {
+	private static Optional<JobID> getJobId(CommandLine commandLine) throws FlinkParseException {
```

Comment:
I think it would also be ok to simply return a nullable `JobID` since it's an internal method.

Reply:
Done